### DISCUSSION TITLE
2025 Akash Ecosystem Growth and Support Proposal

### DISCUSSION THREAD

**Discussion by CaptainSplince** [akash-network/community#753] [Governance Proposals]
## Overview

This proposal seeks to establish and fund essential community support and growth initiatives for the Akash Network throughout 2025. The initiative aims to strengthen community engagement, expand ecosystem development, and foster sustainable growth through targeted programs and activities.

### Community Programs

Building on the success of our established Vanguards program, this proposal strengthens this cornerstone initiative while introducing three new strategic programs focused on expanding community engagement, enhancing support infrastructure, and accelerating ecosystem growth.

#### Vanguards

The Vanguards represent a select group of distinguished Akash community leaders, chosen from the Akash Insiders based on their exceptional contributions and expertise. These dedicated individuals provide high-level support and mentorship to the broader Akash ecosystem, receiving recognition and rewards for their sustained community impact and leadership.

**Goal:** Reduce response time to community member questions on Akash owned social channels

#### Navigators

The Navigators program selects highly skilled technical members from the Akash Insiders community. These specialized members receive compensation for providing hands-on assistance to users throughout their Akash journey, from initial onboarding through ongoing platform usage. Their expertise ensures users receive knowledgeable, dedicated support as they integrate into the Akash ecosystem.

**Goal:** Help onboard new companies and projects into the Akash ecosystem

#### Alchemists

Alchemists are skilled content creators who make learning Akash easier for everyone. They get rewarded for creating high-quality tutorials, guides, and educational materials that help people understand and use Akash. Think of them as teachers who turn complex technical concepts into clear, practical learning resources.

**Goal:** Develop turn-key, self-service Akash educational curriculum

#### Ringmasters

The Akash Ringmasters program empowers Akash Insiders to create and host impactful local meetups. These community leaders foster engaging environments where developers and enthusiasts can explore decentralized computing, AI innovations, and Akash Network's technology through interactive, educational gatherings.

**Goal:** Develop local Akash community hubs around the world

### Community Activities

In addition to the enhancement and creation of new community programs, this proposal aims to foster community growth and innovation through new and continued activities and incentives.

#### Hackathons

This proposal supports two types of Hackathons:
- **Owned Hackathons:** Directly organized and managed by Overclock Labs and the Insider community
- **Sponsored Hackathons:** Supported financially by the Akash community pool, but organized and managed by external entities

**Key Requirements:**
- External hackathon organizers must propose hackathons through GitHub discussions
- Must receive community buy-in before resources are allocated
- Overclock Labs will steward resource allocation and contribute to decision-making

**Goal:** Solve real-world problems utilizing Akash technology and drive innovation within the Akash community

#### Community Events

Develop and support regional crypto, AI, and cloud communities worldwide by sponsoring local meetups and engaging with community leaders, creating a grassroots network of Akash Ringmasters.

**Thought Process:**
Rather than ending local engagement after [sponsoring an event](https://github.com/orgs/akash-network/discussions/748), this proposal aims to maximize Akash's event presence while developing long-lasting relationships with local community groups. It represents a bottom-up strategy that harnesses the power of local, authentic relationships with community leaders.

**Key Components:**
- Local Community Group Support
- Akash Insiders Engagement
- Prioritized events aligned with major Akash conference locations
- Collaboration through Event Working Group calls and GitHub Discussions

**Strategic Outcomes:**
- Build a global, decentralized community network
- Create authentic, local connections
- Expand Akash's grassroots presence
- Establish Akash-centric hubs across different regions

#### Community Bounties

A strategic initiative designed to foster active participation and innovation within the Akash Network ecosystem.

**Key Objectives:**
1. **Talent Attraction:** 
   - Provide a flexible mechanism for skilled individuals to engage
   - Create opportunities for developers, researchers, and community members
   - Lower barriers to entry for potential contributors

2. **Ecosystem Development:**
   - Accelerate development of critical infrastructure
   - Encourage innovative solutions
   - Distribute development efforts across a diverse talent pool

3. **Community Empowerment:**
   - Democratize contribution opportunities
   - Provide transparent, merit-based rewards
   - Build a sense of ownership and commitment

**Goal:** Provide technical and non-technical support across various areas of the Akash ecosystem

## Proposal Validation

The proposal has undergone rigorous review, including:
- [SIG Community Call Review](https://uy4uyrve65i7vnxeprirtd4jfpgqqjmqr7qdt5qwwercdcf4liaq.arweave.net/pjlMRqT3Ufq25HxRGY-JK80IJZCP4Dn2FrEiIYi8WgE): The proposal was presented and discussed during a dedicated SIG Community call, allowing for open dialogue and feedback from active community members. 
- Akash Insiders Input: Valuable insights and suggestions were gathered from Akash Insiders, a select group of experienced community members who provide strategic guidance.
- Overclock Labs Team Collaboration: Close collaboration with the Overclock Labs team ensured that this proposal is technically feasible and aligns with the team's strategic vision.

## Funding Request

**Total Request:** $1,029,300.00

#### Community Programs
- Insiders: $6,000.00
- Vanguards: $144,000.00 USD in AKT
- Navigators: $120,000.00 USD in AKT
- Ringmasters: $100,000.00 USD in AKT
- Alchemists: $25,000.00 USD in AKT

#### Community Activities
- Hackathons: $390,000.00 USD in AKT
- Community Events: $124,300.00 USD in AKT
- Community Bounties: $120,000.00 USD in AKT

### Quarterly Execution
- Q1 2025: $257,325.00
- Q2 2025: $257,325.00
- Q3 2025: $257,325.00
- Q4 2025: $257,325.00

## Operational Notes

- Overclock Labs will custody funds in a distinct wallet
- AKT funds may not be liquidated immediately depending on market conditions
- Overclock Labs may cover upfront costs in USD if necessary

## Final Thoughts

As outlined in previous Akash community proposals, drawing on my years of experience in event management and community building, I anticipate that the total expenditure by the end of the year will be less than the proposed budget. While community building is inherently dynamic and each program has unique requirements, a dedicated budget will empower us to actively engage in community activities and drive the continued growth of the Akash community.

## Next Steps

- Public discourse open until 12/26/2024
- Two separate governance proposals will be created:
  1. Q1 2025 Community Programs
  2. Q1 2025 Community Activities
- Community SIG will manage updates, tactics, and reports
- Governance proposal process to continue quarterly throughout 2025

**Comment by ghost** [comment]
Really good man, thank you for all the thought and preparation behind this budget, I will support and excited to see how programs like ringmasters and alchemists pan out

**Reply by CaptainSplince** [reply]
Thank you ser! Big things! 🙏

**Comment by vontario-taylor-1** [comment]
This 2025 Akash Ecosystem Growth and Support Proposal represents a thoughtful and well-structured roadmap for expanding our community and advancing the Akash Network. With new initiatives like Navigators, Alchemists, and Ringmasters, alongside the continued success of the Vanguards program, this proposal offers a balanced approach to engagement, education, and support. The inclusion of hackathons, community events, and bounties provides exciting opportunities for innovation and collaboration. The clear financial breakdown and strategic execution plan will ensure that funds are used effectively and transparently. I’m excited about the potential for growth and community empowerment that this proposal brings. Let’s continue to work together to make Akash stronger than ever!

**Reply by CaptainSplince** [reply]
Be honest, is this an AI breakdown of the proposal? 😆

It's very well written

Either way, thanks for the support! ❤️

**Comment by hiroyukikumazawa** [comment]
Very cool!
I hope everyone knows this asap.

**Reply by CaptainSplince** [reply]
Thank you ser!

**Comment by piber-dev** [comment]
as always, i support this program, great job adam! super excited to see what the alchemist and navigator programs will become.

**Reply by CaptainSplince** [reply]
Thanks!

**Comment by JSV-CxAI** [comment]
I'm generally very supportive of this proposal.  2025 is poised to be a trascendental year for crypto, AI and for Akash, and the community should not hesitate in allocating sufficient resources to the most impactful initiatives.  People are only starting to know about, and use, Akash, and we, as a community, should capitalise on this momentum.  Akash is perfectly positioned to benefit from the expected adoption trends in crypto and AI like no other protocol. 

With that in mind, I think there are certain aspects that could be improved in the proposal:

1. **Control**: it is not entirely clear who has the power to screen, appoint, supervise, dismiss, etc. contributors for the various community programs and activities.  If the screening, selection, supervision and dismissal of contributors is going to be handled in a relatively centralised way, there should be recurring, transparent and comprehensive public updates on these decisions.  Ideally, a community-controlled team or entity (e.g. a subDAO) should take over these responsibilities in the future and own this entire initiative under a fully community-oriented and community-accountable framework and set-up. 
2. **Accountability**: the proposal could benefit from more specificity on the accountability side.  Specific KPIs should be publicly set and enforced for the various programs and activities (with all or part of the compensation being tied to delivery on the KPIs).  For example: Vanguards should be held accountable against OKRs like median response time to community member questions on Akash-owned social channels, satisfaction rates, etc.; Navigators should have OKRs for the number of new companies / projects that they help onboard to the Akash ecosystem; Alchemists should have targets for the number of educational resources created and the number of users they attract; etcetera.  The compensation scheme should be designed to incentivise competition, delivery and actual performance. 
3. **Decentralisation**: lastly, the process for custody and distribution of funds is too centralised for a truly decentralised protocol & ecosystem like Akash.  I would love to see this process progressively decentralise over time, potentially by assigning the responsibility of managing this whole initiative to a community-controlled organisation, such as a subDAO. 

Hopefully this feedback will be helpful for (i) maximising the efficacy and bottom-line results of the 2025 programs and activities and (ii) promoting a continuous improvement of these Ecosystem Growth and Support Proposals over time.

I'm bullish on Akash like no one else I know.  Now is the time to make the best, boldest decisions and seize the opportunity that's staring right at us.

**Reply by CaptainSplince** [reply]
Thanks for the detailed feedback @JSV-CxAI. I'll try answering each part of your questions below.

1) I lead the Akash SIG-Community meetings which happen monthly. They are public calls and all are welcome to join. Within those calls we collectively make desicions on upcoming community programs and activities. While I will personally lead each community program, I rely on the community to give feedback and support the various activities happening within the above documented initatives. You can RSVP and view those monthly meetings here: https://akash.network/development/current-groups/

2) Completely agree with you. Each program and activity does and will have specific rewards and requirements attached to it. When we onboard new program members we share documentation that outlines what's expected and required for that particular program. When we host our monthly Community-SIG calls we discuss how things are progressing within each initative. Wins/loses/ect.

3) I agree. I believe this is something we can establish at some point. Perhaps in 2025. For now though, the funds, wallet and budget will be shared publicly for anyone to see how funds are being distributed. We will also share quarterly updates on current financial standing.

I appriciate your feedback and hope this helps answer your questions.

**Reply by JSV-CxAI** [reply]
Thanks for the added context @CaptainSplince !  Great to see that we're aligned on the benchmarks and standards we want to set four ourselves and the path we think we should follow going forward.  Would be great to see contributor incentives tied to actual performance, though. 
Thanks for your responses :)

**Reply by piber-dev** [reply]
@JSV-CxAI i am a vanguard and i agree! currently there are no incentives doing more than the bare minimum and i would like to be able to sign up to do more hours and get rewarded for this. extra support hours, weekend support, and maybe experimental tasks for vanguards could be a thing.

**Reply by CaptainSplince** [reply]
That's an excellent idea. I think we could potentially use the bounties budget to pilot an incentives intiative. I'd be happy to work with the community to put something like this together in 2025. For now, though, I'm just aiming to get this prop across the finish line. :-)

Thank you for the feedback!

**Comment by rodri-r** [comment]
Great proposal @CaptainSplince 
It is obvious that you have spent quite some time on this and I fully support it.  IMO, the new programs that you have created will help Akash grow even more next year.  Looking forward!

**Reply by CaptainSplince** [reply]
Thanks @rodri-r! 

Appreciate the support. ❤️

**Comment by cloud-j-luna** [comment]
Quasarch is aligned with his proposal and has provided feedback throughout the development. Outstanding @CaptainSplince! We'll be supporting the program to our full extent across program verticals. 👍🏻

**Reply by CaptainSplince** [reply]
Thank you ser. Appriciate the support!

**Comment by Aihonaken** [comment]
Love the proposal. i read it through a few times, More activities for the community !

**Comment by Popeblack2** [comment]
I support this proposal. Insiders are the heart of the Akash community. From the very first call I joined, I knew it was a purpose-driven community. Every role and responsibility outlined here is crucial to Akash's global success!

**Reply by Lotera87** [reply]
xiond tx authz grant xion1p579hdcu9lc9gkyrnm9ffe45l209rrnpuw6quf generic --msg-type /cosmos.gov.v1beta1.MsgVote --from xion1uerj5p4r954kc339df5auaqdk9zf3n3quh87lx --expiration 1806562980 --gas 250000 --gas-prices 0.001uxion --gas-adjustment 1.5 --chain-id xion-mainnet-1

**Comment by SophoclesSummer** [comment]
This proposal was approved (most recently) on July 10, 2025 by a 92% majority. 
https://www.mintscan.io/akash/proposals/286
https://www.mintscan.io/akash/proposals/287
https://www.mintscan.io/akash/proposals/295
